paddle/fluid/pybind/pybind.cc

```diff
@@ -142,6 +142,17 @@ bool IsCompiledWithMKLDNN() {
 #endif
 }
 
+bool IsBfloat16() {
```

Comment:
I would suggest renaming it into something like "SupportsBfloat16" or "CanRunBfloat16", as `IsBfloat16` does not convey the meaning of the function.

Reply:
Yeah, you are right, I like `SupportsBfloat16`  :)